htdocs/view/layout/header.php

```diff
@@ -104,19 +104,32 @@
   </li>
 
   <li>
-		<?php echo link_to(path("logout", "home"), "<i class=\"fa fa-fw fa-power-off\" style=\"color:#fff;\"></i>") ?>
+    <?php echo link_to(path("logout", "home"), "<i class=\"fa fa-fw fa-power-off\" style=\"color:#fff;\"></i>") ?>
   </li>
 
   <!-- Modal : the user can choose the wave to ask for subsidies (make a request) -->
   <?php
     if (isset($_GET["prefix"]) && $_GET["prefix"] == "binet" && has_editing_rights($binet, $term)) {
         ob_start();
-        $waves_for_modal = select_waves(array("state" => "submission"), "submission_date", false);
+        $waves_for_modal = select_waves(array("state" => array("submission", "deliberation")), "submission_date", false);
```

Comment:
Hmm oui ok. 
Ca me rapelle mon oral de Vérif, ou X.Rival m'avait dit que ce que j'avais fait avec le projet, ca me permettait par exemple de créer un langage de requêtes.
Ca serait par exemple :
select_operations("amount > 0 & state IN submission, deliberation ")
Mais bon.
Je ne pense pas qu'il y ait beaucoup de gain, c'est lisible avec le in dans l'array.